alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+ENV RAILS_ENV=test                  \
+    PGBASE=/var/lib/postgresql      \
+    PGDATA=/var/lib/postgresql/data
```

Comment:
I don't support environment variables can refer to others set just prior, eg: `PGDATA=$PGBASE/data`